applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -315,28 +396,53 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 		LOG_ERR("Failed to set security to L2: %d", ret);
 	}
 
-	/* TODO: Channel nr is 0 for now, to be changed with dual CIS */
-	ble_vcs_discover(conn, 0);
+	if (!ble_acl_gateway_all_links_connected()) {
+		ble_acl_start_scan();
+	} else {
+		bt_le_scan_stop();
```

Comment:
Hasn't scanning already been stopped in the device found function?

Reply:
That would make sense at least